docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -220,19 +220,23 @@ To simulate gravitational accelerations that don't align with the vertical (`z`)
 we wrap the buoyancy model in
 `Buoyancy()` function call, which takes the keyword arguments `model` and `gravity_unit_vector`,
 
-```jldoctest buoyancy
+```jldoctest buoyancy; filter = r".*tartarus.*"
```

Comment:
If you still have trouble then you can try 
```suggestion
```jldoctest buoyancy; filter = [r".*┌ Warning.*", r".*└ @ Oceananigans.*"]
```
which had worked, I believe, previously


```